src/components/OptionsTiles/OptionTiles.Overview.stories.mdx

```diff
@@ -0,0 +1,556 @@
+        },
+        {
+          id: 'vanilla_multi',
+          value: 'vanilla',
+          label: 'Vanilla',
+        },
+      ];
+      return (
+        <OptionTiles
+          name="multi"
+          value={value}
+          onChange={handleSetValue}
+          options={options}
+          isMulti
+        />
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Hidden Input
+
+You can render tiles without a radio/checkbox. Note that the input elements are still underlying the component, but this will limit the visual indication
+to the selected states on the tile itself. This is likely best when paired with custom content on the tile.
+
+<Canvas isExpanded>
+  <Story name="Hidden Input">
+    {() => {
+      const [value, setValue] = useState([]);
+      const options = [
+        {
+          id: 'chocolate_hidden',
+          value: 'chocolate',
+          label: 'Chocolate',
+        },
+        {
+          id: 'strawberry_hidden',
+          value: 'strawberry',
+          label: 'Strawberry',
+        },
+        {
+          id: 'vanilla_hidden',
+          value: 'vanilla',
+          label: 'Vanilla',
+        },
+      ];
+      return (
+        <OptionTiles
+          name="hidden"
+          value={value}
+          onChange={event => setValue(event.target.value)}
+          options={options}
+          hideInput
+        />
+      );
+    }}
+  </Story>
+</Canvas>
+
+## Direction
+
+Set the direction (flex) of the tiles. This uses the same `direction` prop as the `Box` component and as such can be responsive.
+
+<Canvas isExpanded>
+  <Story name="Direction">
+    {() => {
+      const [valueHorizontal, setValueHorizontal] = useState([]);
+      const [valueVertical, setValueVertical] = useState([]);
+            const optionsVertical = [
+        {
+          id: 'chocolate_direction_vertical',
+          value: 'chocolate',
+          label: 'Chocolate',
+        },
+        {
+          id: 'strawberry_direction_vertical',
+          value: 'strawberry',
+          label: 'Strawberry',
+        },
+        {
+          id: 'vanilla_direction_vertical',
+          value: 'vanilla',
+          label: 'Vanilla',
+        },
+      ];
+      const optionsHorizontal = [
+        {
+          id: 'chocolate_direction_horizontal',
+          value: 'chocolate',
+          label: 'Chocolate',
+        },
+        {
+          id: 'strawberry_direction_horizontal',
+          value: 'strawberry',
+          label: 'Strawberry',
+        },
+        {
+          id: 'vanilla_direction_horizontal',
+          value: 'vanilla',
+          label: 'Vanilla',
+        },
+      ];
+      return (
+        <>
+          <p className="m-bottom-md font-size-lg font-weight-bold">Vertical</p>
+          <OptionTiles
+            name="directionVertical"
+            value={valueVertical}
+            onChange={event => setValueVertical(event.target.value)}
+            options={optionsVertical}
+            direction="column"
+          />
```

Comment:
Since the other stories are column by default, don't think we need this example.